src/stactools/viirs/commands.py

```diff
@@ -18,39 +24,165 @@ def create_viirs_command(cli: Group) -> Command:
+        "-c",
+        "--create-cogs",
+        is_flag=True,
+        help="Convert the H5 subdatasets into COGs",
+        default=False,
+    )
+    @click.option(
+        "-d",
+        "--densify_factor",
+        help="Factor by which to densify the Item geometry",
+        type=int,
+    )
+    @click.option(
+        "-f", "--file-list", help="File containing list of subdataset COG HREFs"
+    )
+    def create_item_command(
+        infile: str,
+        outdir: str,
+        antimeridian_strategy: str,
+        create_cogs: bool,
+        densify_factor: Optional[int] = None,
+        file_list: Optional[str] = None,
+    ) -> None:
+        """Creates a STAC Item based on an H5 VIIRS data file and, if it exists,
+        the corresponding XML metadata file.
+
+        \b
+        Args:
+            infile (str): The source H5 file.
+            outdir (str): Directory that will contain the STAC Item.
+            antimeridian_strategy (str, optional): Choice of 'normalize' or
+                'split' to either split the Item geometry on -180 longitude or
+                normalize the Item geometry so all longitudes are either
+                positive or negative. Default is 'split'.
+            create_cogs (bool, optional): Flag to create COGS from the
+                subdatasets in the H5 file. The COGs will saved alongside the H5
+                file. COGs will not be created if the file_list option is also
+                supplied.
+            file_list (str, optional): Text file containing one HREF per line.
+                The HREFs should point to subdataset COG files.
+        """
+        strategy = Strategy[antimeridian_strategy.upper()]
+
+        hrefs = None
+        if file_list:
+            with open(file_list) as file:
+                hrefs = [line.strip() for line in file.readlines()]
+        elif create_cogs:
+            h5dir = os.path.dirname(infile)
+            hrefs = cog.cogify(infile, h5dir)
+
+        item = stac.create_item(
+            infile,
+            cog_hrefs=hrefs,
+            antimeridian_strategy=strategy,
+            densify_factor=densify_factor,
+        )
+        item_path = os.path.join(outdir, f"{item.id}.json")
+        item.set_self_href(item_path)
+        item.make_asset_hrefs_relative()
+        item.validate()
+        item.save_object()
 
         return None
 
-    @viirs.command("create-item", short_help="Create a STAC item")
-    @click.argument("source")
-    @click.argument("destination")
-    def create_item_command(source: str, destination: str) -> None:
-        """Creates a STAC Item
+    @viirs.command("create-collection", short_help="Create a STAC Collection")
+    @click.argument("INFILE")
+    @click.argument("OUTDIR")
+    @click.option(
+        "-c",
+        "--create-cogs",
+        is_flag=True,
+        help="Convert the H5 subdatasets into COGs",
+        default=False,
+    )
+    @click.option(
+        "-a",
+        "--antimeridian_strategy",
```

Comment:
```suggestion
        "--antimeridian-strategy",
```

Reply:
man, i was consistent with the underscores. 🙂  Done.